index.html

```diff
@@ -4,9 +4,307 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
+    <link href="./style.css" rel="stylesheet">
+    <title>Microsoft</title>
 </head>
-<body>
+
+<body >
+    <div class="mx-48">
```

Comment:
@ktscates it would be better if you use `width `more than using `mx`
for example like this
```suggestion
    <div class="w-10/12 mx-auto">
```